docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+services:
+  node:
+    image: node:21.5-bullseye-slim
+    container_name: e-motion-web
+    working_dir: /e-motion-web
+    volumes:
+      - .:/e-motion-web
+    restart: unless-stopped
+    tty: true
+    # ports:
+    #   - ${DOCKER_APP_HOST_PORT:-3000}:3000
+    command: bash -c "npm install && npm run dev"
```

Comment:
I do not think so it is good way..
I wanna run `make dev`(`npm run dev`) and then app should start.
What if I wanna enter to container and run other command - `npm update`?

Reply:
you can enter by running ```make shell```. I think it is convenient to have it running with one command.